### PR TITLE
Update management plane event grid

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Testing out Management Plane Restructuring.

### CODE REVIEW

**Comment by chidozieononiwu** [comment]
@Azure/azure-sdk-eng

File: sdk/AzSdk.reference.props [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (weshaggard) [inline sdk/AzSdk.reference.props]: Why is this needed?

File: sdk/AzSdk.test.reference.props [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```
Comment (weshaggard) [inline sdk/AzSdk.test.reference.props]: Why is this needed?

File: sdk/eventgrid/Microsoft.Azure.Management.EventGrid/src/Microsoft.Azure.Management.EventGrid.csproj [hunk 3]
```diff
@@ -15,7 +15,8 @@
     <TargetFrameworks>$(SdkTargetFx)</TargetFrameworks>
   </PropertyGroup>
 
-  <!-- Please do not move/edit code below this line -->
-  <Import Condition=" Exists('$([MSBuild]::GetPathOfFileAbove(AzSdk.RP.props))') " Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.RP.props'))" />
-  <!-- Please do not move/edit code above this line -->
+  <PropertyGroup>
```
Comment (weshaggard) [inline sdk/eventgrid/Microsoft.Azure.Management.EventGrid/src/Microsoft.Azure.Management.EventGrid.csproj]: For the management libraries I think we should keep the import. @shahabhijeet do you still generate those AzSdk.RP.props files?

File: sdk/eventgrid/Microsoft.Azure.Management.EventGrid/EventGrid.sln [hunk 4]
```diff
@@ -3,9 +3,9 @@ Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 15
 VisualStudioVersion = 15.0.27130.2020
```
Comment (weshaggard) [inline sdk/eventgrid/Microsoft.Azure.Management.EventGrid/EventGrid.sln]: @shahabhijeet are the solutions files needed or even used for the auto-generated management libraries?

File: sdk/AzSdk.reference.props (hunk 1 above)
Comment (chidozieononiwu) [reply sdk/AzSdk.reference.props]: It contains $(SdkTargetFx) which is being used in the managementplane .csproj as TargetFramework

File: sdk/AzSdk.test.reference.props (hunk 2 above)
Comment (chidozieononiwu) [reply sdk/AzSdk.test.reference.props]: The test Projects import it

Comment (weshaggard) [reply sdk/AzSdk.test.reference.props]: This should still be coming from the management toolset and not be added here.

File: sdk/AzSdk.reference.props (hunk 1 above)
Comment (weshaggard) [reply sdk/AzSdk.reference.props]: Again I think this is from the management toolset and shouldn't be here. These projects shouldn't be getting built from our pipelines they should still be getting built from the mgmt pipelines but you might need to make some adjustments to ensure that still happens.

Comment (chidozieononiwu) [reply sdk/AzSdk.reference.props]: Ok, I'll do that.
